external_services/templates/external_services/_launch.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
+{% load external_services %}
+{% load course %}
+{% load base %}
+{% comment %}
+  expected parameters:
+    service - LinkService
+	site - service front page
+	[parameters] - key/value pairs for post form, can be ommitted for get
+    parameters_hash - hash of form fields
+	[exercise] - exercise that holds this link
+	is_course_staff
+{% endcomment %}
+{% comment %}
+static/js/external_launcher.js defines a jQuery plugin that should be used with
```

Comment:
I think this comment is outdated since this PR changes the JS file path and how the JS is activated.

Reply:
It's still required. Path is correct inside django app (external_services). Though,. the comment about jQuery isn't valid anymore.